src/PersonaListController.js

```diff
@@ -3,7 +3,7 @@
  * Created by Chin on 08-Apr-17.
  */
 var PersonaListController = (function () {
-    function PersonaListController($scope) {
+    function PersonaListController($scope, $timeout) {
```

Comment:
Sorry, only saw this now. It's not used, it was part of an early change that I eventually removed; feel free to delete it.